TFSIntegration/Script.ps1

```diff
@@ -0,0 +1,1609 @@
+
+        public static readonly String NO_SUCH_RUN_WAS_FOUND = "Could not find run {0} of {1} schedule!";
+        public static readonly String NO_SUCH_RUN_ITEM_WAS_FOUND = "Could not find run item {0} of {1} schedule!";
+        public static readonly String NO_SUCH_RUN = "No such run {0} !";
+
+        public static readonly String REPORT_FILE_NOT_FOUND = "Couldn't find report file! Wrong path! Press \"help\" button nearby \"report\" textbox! ";
+        public static readonly String REPORT_FILE_CREATION_FAILURE = "Failed to create a report file!";
+
+        public static readonly String CASE_CONSOLE_LOG_SEPARATOR = "----------------------------------------------------------------------------------------";
+        public static readonly String SCHEDULE_CONSOLE_LOG_SEPARATOR = "//////////////////////////////////////////////////////////////////////////////////////";
+
+        public static readonly String CASE_INFORMATION = "Run Item: {0} | Status: {1} | Elapsed: {2}";
+        public static readonly String CASE_STACKTRACE_FORMAT = "{0} - {1}";
+
+        public static readonly String GET_ALL_AVAILABLE_SCHEDULES_URI = "{0}/api/v4/schedules";
+        public static readonly String RUN_SCHEDULE_URI = "{0}/api/v4/schedules/{1}/runNow";
+        public static readonly String STOP_RUN_URI = "{0}/api/v4/run/{1}/stop";
+        public static readonly String GET_RUN_STATUS_URI = "{0}/api/v4/run/{1}/status";
+        public static readonly String GET_RUN_ITEMS_IDS_URI = "{0}/api/v4/run/{1}/runItemIds";
+        public static readonly String GET_RUN_ITEM_URI = "{0}/api/v4/runItems/{1}";
+        public static readonly String GET_RUN_ITEM_KEYFRAMES = "{0}/api/v4/runItems/{1}/keyframes";
+
+        public static readonly String INVALID_SCHEDULES = "INVALID SCHEDULES";
+        public static readonly String PLUGIN_NAME = "Leapwork Integration";
+
+        public static readonly String NO_SCHEDULES = "No Schedules to run! All schedules you've selected could be deleted. Or you simply have forgotten to select schedules after changing controller address;";
+
+        public static readonly String PLUGIN_SUCCESSFUL_FINISH = "Leapwork for TFS  plugin  successfully finished!";
+        public static readonly String PLUGIN_ERROR_FINISH = "Leapwork for TFS plugin finished with errors!";
+
+        public static readonly String CONTROLLER_RESPONDED_WITH_ERRORS = "Controller responded with errors! Please check controller logs and try again! If does not help, try to restart controller.";
+        public static readonly String PLEASE_CONTACT_SUPPORT = "If nothing helps, please contact support https://leapwork.com/chat and provide the next information:  \n1.Plugin Logs  \n2.Leapwork and plugin version  \n3.Controller logs from the moment you've run the plugin.  \n4.Assets without videos if possible.  \nYou can find them {Path to Leaptest}/LEAPTEST/Assets  \nThank you";
```

Comment:
what is Leaptest?

Reply:
Updated Leaptest to Leapwork in latest commit.